src/integration/pages/articles/afrique/canonical.test.js

```diff
@@ -4,9 +4,15 @@
  */
 
 import runGistTests from '../gistTests';
+import runMediaPlayerCaptionTests from '../mediaPlayerCaption';
+import runMediaPlayerPlaceholderTests from '../mediaPlayerPlaceholderTestsCanonical';
+import runImageCopyrightBBC from '../imageCopyrightNonBBC';
 
 describe('Canonical', () => {
   describe(pageType, () => {
     runGistTests();
+    runMediaPlayerCaptionTests(); // MIGHT NEED TO ADD LTR VERSION
+    runMediaPlayerPlaceholderTests(); // MIGHT NEED TO ADD LTR VERSION
```

Comment:
We can remove these comments now given that we have an LTR integration test

```suggestion
    runMediaPlayerCaptionTests();
    runMediaPlayerPlaceholderTests(); 
```